src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -480,10 +480,10 @@ def get_kubernetes_distro(configuration):  # Heuristic
     api_instance = kube_client.CoreV1Api(kube_client.ApiClient(configuration))
```

Comment:
I would suggest we figure out the distro and infra values in a single list_node() call

Reply:
Updated.